viadot/tasks/cloud_for_customers.py

```diff
@@ -1,46 +1,132 @@
-from prefect import task
+from prefect import task, Task
 import pandas as pd
 from ..sources import CloudForCustomers
 from typing import Any, Dict, List
+from prefect.utilities.tasks import defaults_from_attrs
 
 
-@task
-def c4c_report_to_df(report_url: str, env: str = "QA", skip: int = 0, top: int = 1000):
-    final_df = pd.DataFrame()
-    next_batch = True
-    while next_batch:
-        new_url = f"{report_url}&$top={top}&$skip={skip}"
-        chunk_from_url = CloudForCustomers(report_url=new_url, env=env)
-        df = chunk_from_url.to_df()
-        final_df = final_df.append(df)
-        if not final_df.empty:
-            df_count = df.count()[1]
-            if df_count != top:
-                next_batch = False
-            skip += top
-        else:
-            break
-    return final_df
-
-
-@task
-def c4c_to_df(
-    url: str = None,
-    endpoint: str = None,
-    report_url: str = None,
-    fields: List[str] = None,
-    params: Dict[str, Any] = {},
-    env: str = "QA",
-    if_empty: str = "warn",
-):
-    cloud_for_customers = CloudForCustomers(
-        url=url,
-        report_url=report_url,
-        endpoint=endpoint,
-        params=params,
-        env=env,
+class C4CReportToDF(Task):
```

Comment:
Add docstrings on what these tasks do. Check out any Prefect task for an example.